packages/amplify-cli-auth/src/commands/login.js

```diff
@@ -39,8 +39,14 @@ export default {
 		'-c, --client-secret [key]': 'A secret key used to authenticate',
 		'-s, --secret-file [path]':  'Path to the PEM key used to authenticate',
 		'--service':                 'Authenticates client secret for non-platform service account',
-		'-u, --username [user]':     'Username to authenticate with',
-		'-p, --password [pass]':     'Password to authenticate with'
+		'-u, --username [user]': {
```

Comment:
we probably need a doc update also: https://docs.axway.com/bundle/Axway_CLI_allOS_en/page/axway_cli_authentication.html

Reply:
I've already sent a doc update request to the docs team.